src/Elliptic/Actions/InitializeSystem.hpp

```diff
@@ -26,15 +26,12 @@ namespace Actions {
  *
```

Comment:
These `Upgrade elliptic executables` type short git messages aren't that helpful. This seems to be doing at least 3 things at once. Can they not be separated out?